custom/v3/command/data/src/main/java/org/apache/fineract/v3/command/data/CommandResponse.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.fineract.v3.command.data;
+
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import java.io.Serial;
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.UUID;
+import lombok.Data;
+import lombok.experimental.FieldNameConstants;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
+@FieldNameConstants
+public class CommandResponse<T> implements Serializable {
+
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    private UUID requestId;
+    private String tenantId;
+    private Long userId;
+    private String errorCode;
+    private String errorMessage;
+    private Instant createdAt;
+
+    @JsonUnwrapped
```

Comment:
I don't think you want to do this... JsonUnwrapped is flattening out the attributes of an embedded object.

Reply:
Yes, I used it to return the same response structure as the upstream API does.
It's been removed now.

> [!NOTE]
>
> The difference between responses:
>
> - **Current Response:**
> ```json
> {
>     // other attributes
>     "data": {
>         "resourceId": 1
>         // data attributes
>     }
> }
> ```
> - **The Upstream Response:**
> ```json
> {
>     "resourceId": 1
> }
> ```

Reply:
Yes, we want the first thing! Remove that Unwrapped thing please... and yes, the data structure looks a little bit different than upstream... but that's why we do this proof of concept and cleanup!